surfnperf.js

```diff
@@ -356,6 +356,26 @@
     return this.duration('navigationStart', 'loadEventEnd');
   };
 
+  /**
+   * Time to first paint (amount of time that lapses before some useful content is diplayed indicating that the browser is not stuck and is still working)
```

Comment:
Let's change this to `Time to first paint (browser-reported time for the first pixels to be rendered on the page)`